Missionframework/modules/11_permission/fnc/fn_permission_openDialog.sqf

```diff
@@ -53,7 +59,9 @@ private _player = "";
     _index = _ctrlPlayerList lbAdd (name _x);
     _ctrlPlayerList lbSetData [_index, getPlayerUID _x];
     _player = _x;
-    _playerList deleteAt (_playerList findIf {((_x select 1) isEqualTo (getPlayerUID _player))});
+    _playerList deleteAt (_playerList findIf {
+        ((_x select 1) isEqualTo (getPlayerUID _player))
+    });
 } forEach (allPlayers - (entities "HeadlessClient_F"));
```

Comment:
You could use [CBA_fnc_players](https://cbateam.github.io/CBA_A3/docs/files/common/fnc_players-sqf.html#CBA_fnc_players) here.

Reply:
Maybe I'm missing something obvious currently (as the CBA function uses allUnits + allDeadMen instead of allPlayers), but the CBA function is ~14x slower. The CBA function does:
`(allUnits + allDeadMen) select {isPlayer _x && {!(_x isKindOf "HeadlessClient_F")}}`

```
Result:
0.0267 ms

Cycles:
10000/10000

Code:
test = [] call CBA_fnc_players;
```

```
Result:
0.002 ms

Cycles:
10000/10000

Code:
test = allPlayers - (entities "HeadlessClient_F")
```